Build failure: rPackages.ROracle

### Steps To Reproduce

Steps to reproduce the behavior:
1. `nix shell nixpkgs#rPackages.ROracle`
### Build log

```
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/grnzing5w9d8xj210mv598i2ddsdsys6-ROracle_1.3-1.1.tar.gz
source root is ROracle
setting SOURCE_DATE_EPOCH to timestamp 1636546939 of file ROracle/MD5
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "configurePhase" }
configuring
@nix { "action": "setPhase", "phase": "buildPhase" }
building
@nix { "action": "setPhase", "phase": "checkPhase" }
running tests
@nix { "action": "setPhase", "phase": "installPhase" }
installing
* installing *source* package 'ROracle' ...
** package 'ROracle' successfully unpacked and MD5 sums checked
** using staged installation
configure: error: OCI libraries not found
ERROR: configuration failed for package 'ROracle'
* removing '/nix/store/dy3aw6k2nl44fd6czmsysc2y5f3a2hax-r-ROracle-1.3-1.1/library/ROracle'
```

### Additional context
I tried to find which deps it's looking for, but couldn't figure out which Oracle-related lib it might be. There's a chance that it's not worth the effort, as r2u also blacklists it as broken:

https://github.com/eddelbuettel/r2u/blob/c14a15dde35577be56767f9c945eb2149882265c/README.md?plain=1#L250

Install instructions on CRAN
https://cran.r-project.org/web/packages/ROracle/INSTALL

### Notify maintainers

No maintainer, or maybe @jbedo ?
### Metadata

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.29, NixOS, 23.05 (Stoat), 23.05.20230520.3005f20`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.3`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

--- a/pkgs/development/r-modules/default.nix
+++ b/pkgs/development/r-modules/default.nix
@@ -1281,6 +1281,13 @@ let
       '';
     });
 
+    ROracle = old.ROracle.overrideAttrs (attrs: {
+      configureFlags = [
+        "--with-oci-lib=${pkgs.oracle-instantclient.lib}/lib"
+        "--with-oci-inc=${pkgs.oracle-instantclient.dev}/include"
+      ];
+    });
+
     sparklyr = old.sparklyr.overrideAttrs (attrs: {
       # Pyspark's spark is full featured and better maintained than pkgs.spark
       preConfigure = ''

